services/manager/lib/io/http/index.js

```diff
@@ -13,17 +13,18 @@ const directoryListing = ({ fileList, directory }, callback) => {
   callback(false, JSON.stringify(listing));
 };
 
-const mountApp = ({ server, name, path }) => {
-  const mountAt = `/${name}`;
+const mountApp = ({ server, name, path, index }) => {
+  const httpPath = `/${name}`;
 
-  const serveStatic = express.static(
-    path,
-    { index: ['index.html', 'index.htm', 'index.js'] }
-  );
+  const serveStatic = express.static(pathLib.join(path, 'src'), { index });
+  server.use(httpPath, serveStatic);
 
-  const serveDirectory = serveIndex(path, { template: directoryListing });
+  const serveModules = express.static(pathLib.join(path, 'node_modules'));
+  server.use(pathLib.join(httpPath, 'modules'), serveModules);
 
-  server.use(mountAt, serveStatic, serveDirectory);
+  const serveAssets = express.static(pathLib.join(path, 'assets'));
+  const listAssets = serveIndex(pathLib.join(path, 'assets'), { template: directoryListing });
```

Comment:
Ah, interesting, so `app-name/assets` enables a JSON listing of the directory but not in `app-name/src`?

That's a really neat distinction. 👍